### ISSUE TITLE
obelisk master branch: config/common/route didn't take affect 

### ISSUE THREAD

**Issue by clojurians-org** [obsidiansystems/obelisk#473]
i want to customize the port from obelisk server by configure the  config/common/route.
but it didn't work from master brach commit (de99fc6178fe64f6ea3176746e9c273c92fb8e45)

the master branch have two version: 
ca6edae87909d0d3c31f6c4a89a2f70bcb209924[old]:
```haskell
_backend_run :: ((R backendRoute -> Snap ()) -> IO ()) -> IO ()
```
de99fc6178fe64f6ea3176746e9c273c92fb8e45[new]
```haskell
_backend_run :: ((R backendRoute -> BackendConfigsT Snap ()) -> BackendConfigsT IO ()) -> BackendConfigsT IO ()
```

and the develop branch switch to the old interface without BackendConfigsT.

so i wonder whether the current master interface is unstable, and is it better to use develop version?

and what's the prefer strategy to process configure for each version?

**Comment by cgibbard** [comment]
Yeah, that's an unfortunate bug in master -- it's looking for that config file in the wrong location in a Map of configs due to some other changes which went through and missed that bit.

I have a branch which will fix that (and does a whole bunch of other stuff, including removing BackendConfigsT again, since it was only ever a nuisance when trying to set up your backend and needing to use things that were written in CPS).

I just need to get it all merged, the PR is a bit of a mess atm. https://github.com/obsidiansystems/obelisk/tree/cg-fullroute

**Comment by JBetz** [comment]
@clojurians-org Are you still having this issue? 

I wasn't able to reproduce this on current master. I.e., changing the port in `config/common/route` worked correctly. I'm guessing @cgibbard's fix got merged at some point, probably in #452.
